torch/serialization.py

```diff
@@ -1269,6 +1269,16 @@ def persistent_load(saved_id):
             f"Received object of type \"{type(f)}\". Please update to Python 3.8.2 or newer to restore this "
             "functionality.")
 
+    start = f.tell()
+    f.seek(0, 2)
+    file_size = f.tell()
+    # Check that file is long enough to contain magic number and
+    # protocol version: 10 bytes for MAGIC_NUMBER, 2 bytes for
+    # PROTOCOL_VERSION, 8 bytes for unpickling operators.
```

Comment:
Could you elaborate on why 8 bytes?

Reply:
Valid test that I tested: 2 bytes for PROTO (start unpickling), 2 bytes for LONG1 and n_bytes, [10 bytes for magic number], 1 byte for STOP, 2 bytes for PROTO, 1 byte for BININT2, [2 bytes for protocol].

Also after protocol there should be 1 byte for STOP (thus, 9 bytes of system info), and after that _sys_info is also expected to be in file, but the main idea of this check is to simply validate that file is long enough to contain constant values that we try to read and check.